pkg/ccl/workloadccl/fixture_test.go

```diff
@@ -115,6 +115,11 @@ func TestFixture(t *testing.T) {
 		t.Fatalf(`%+v`, err)
 	}
 
+	_, err = MakeFixture(ctx, sqlDB.DB, gcs, store, gen)
+	if !testutils.IsError(err, `already exists`) {
+		t.Fatalf(`expected 'already exists' error got: %+v`, err)
```

Comment:
ooc why do you use raw string literals everywhere?

Reply:
so i don't have to think about escaping when i'm reading code /shrug